k2/csrc/util/logging.h

```diff
@@ -0,0 +1,33 @@
+// k2/csrc/util/logging_is_google_glog.h
```

Comment:
filename wrong?

Reply:
This filename is referred from pytorch, and I think it's clear (TF use `default/logging.h` (customized minimal glog) and `google/build_config/logging.h`(glog) to distinguish each other.).